perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py

```diff
@@ -70,12 +72,18 @@
 
 
 # Stress test options.
-CONSISTENCY_LEVEL = 'quorum'
+CONSISTENCY_LEVEL = 'QUORUM'
 REPLICATION_FACTOR = 3
 RETRIES = 1000
 
 CASSANDRA_STRESS = posixpath.join(cassandra.CASSANDRA_DIR, 'tools', 'bin',
                                   'cassandra-stress')
+RESULTS_METRICS = ['op rate', 'partition rate', 'row rate', 'latency mean',
```

Comment:
Could you comment on what these constants are?
Also, prefer tuples for constants.